hal/src/main/native/athena/HALInternal.h

```diff
@@ -8,5 +8,6 @@
 
 namespace hal {
 void ReleaseFPGAInterrupt(int32_t interruptNumber);
+const char* HAL_GetLastError(int32_t* status);
```

Comment:
I think this should be a declaration for `SetLastError()` (defined in ErrorHandling.cpp).